src/index.js

```diff
@@ -198,7 +198,8 @@ class Kronos extends Component {
   }
 
   save(saving) {
-    const { datetime } = this.state
+    let { datetime } = this.state
```

Comment:
Is the dateTime null or undefined? If its undefined we can keep it cont with fallback as Moment()
```suggestion
    const { datetime = Moment() } = this.state
```

Reply:
It's null. So default value will not work.